backend/src/test/java/com/woowacourse/naepyeon/acceptance/MemberAcceptanceTest.java

```diff
@@ -44,53 +48,173 @@ void addDuplicationMember() {
         assertThat(response.statusCode()).isEqualTo(HttpStatus.BAD_REQUEST.value());
     }
 
+    @ParameterizedTest
+    @DisplayName("회원가입 시, 회원이름이 2 ~ 20자 및 한글, 영어, 숫자가 아닌 경우 예외가 발생한다.")
+    @ValueSource(strings = {"0", "012345678901234567891", "+특수문자+"})
+    void failSignUpUsername(String failUsername) {
```

Comment:
```suggestion
    void failSignUpUsername(final String failUsername) {
```

파라미터 부분에도 final 키워드를 적용시키면 좋을 것 같아요!